Client/src/Forms/validateLogin.js

```diff
@@ -1,6 +1,13 @@
+/** [validateLogin.js]
+* Summary. 
+Validating that the login values are real.
+*
+* Description. 
+Showing user that input boxes needs to be filled in orde to make a successful login.
```

Comment:
Typo: "... to be filled in ***orde*** to make successful login"

Reply:
done